preprocess/pca.py

```diff
@@ -16,13 +21,14 @@ def pca(input_prefix: str, pca_config: dict, output_tag: str) -> None:
                   args_dict=pca_config)
 
     @staticmethod
-    def pc_scatterplot(input_prefix: str, output_tag: str) -> None:
+    def pc_scatterplot(tag: str) -> None:
         """ Visualises eigenvector with scatterplot [matrix] """
-        pass
-        # TODO
-        #"plotly_graph".write_html("smth_based_on_output_tag.html")
+        eigenvec = pd.read_table(f'{data_root}/pca/{tag}.eigenvec')[['IID', 'PC1', 'PC2']]
+        eigenvec.index = eigenvec.IID
```

Comment:
why create duplicates? you can send the column to index with eigenvec = eigenvec.set_index('IID') or you can set it as index one while reading in the previous line